src/System.Drawing.Primitives/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1 @@
+user32.dll!GetSysColor
```

Comment:
@hughbe you should be able to delete this file again, now you did your uap change?